<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RulesWindow</class>
 <widget class="QMainWindow" name="RulesWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1110</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1111</width>
      <height>631</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>-40</y>
      <width>661</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="text">
     <string>游戏规则</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>1041</width>
      <height>451</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiLight</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Bahnschrift SemiLight'; font-size:28pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;h3 style=&quot; margin-top:14px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;游戏目标：&lt;/span&gt;&lt;/h3&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;数独是一款经典的数字填充游戏，玩家需要在一个 9x9 的网格中填写数字，使得每一行、每一列以及每个 3x3 宫内都包含 1 到 9 的数字，且每个数字不能重复。玩家可以选择&lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;自动生成初盘&lt;/span&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;或者&lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;人工设置初盘&lt;/span&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;，并且系统会检测人工设置的初盘是否合法。&lt;/span&gt;&lt;/p&gt;
&lt;h3 style=&quot; margin-top:14px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;游戏模式：&lt;/span&gt;&lt;/h3&gt;
&lt;ol style=&quot;margin-top: 0px; margin-bottom: 0px; margin-left: 0px; margin-right: 0px; -qt-list-indent: 1;&quot;&gt;&lt;li style=&quot; font-size:14pt;&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;自动生成初盘&lt;/span&gt;：系统会根据玩家选择的难度自动生成一个随机数独题目，题目包含一定数量的空格（根据难度设定，空格数目不同）。玩家需要在规定时间内填入数字，完成数独解题。&lt;/li&gt;
&lt;li style=&quot; font-size:14pt;&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;人工设置初盘&lt;/span&gt;：玩家可以手动输入数独初盘，通过键盘在 9x9 的网格中设置初始数字。系统将自动检测设置的初盘是否合法，确保每行、每列和每个 3x3 宫内数字不重复。如果初盘不合法，系统会提示玩家重新设置。&lt;/li&gt;
&lt;li style=&quot; font-size:14pt;&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;求解数独&lt;/span&gt;：游戏提供了一个自动求解功能，使用DPLL算法来计算数独的解。如果玩家卡住了，可以点击“求解”按钮查看完整的答案。&lt;/li&gt;&lt;/ol&gt;
&lt;h3 style=&quot; margin-top:14px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;游戏交互功能：&lt;/span&gt;&lt;/h3&gt;
&lt;ul style=&quot;margin-top: 0px; margin-bottom: 0px; margin-left: 0px; margin-right: 0px; -qt-list-indent: 1;&quot;&gt;&lt;li style=&quot; font-size:14pt;&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;检查&lt;/span&gt;：当玩家填写完数独并点击“检查”按钮时，程序会标记出玩家填写正确和错误的数字。正确的答案会以绿色显示，错误的答案会以红色显示。几秒钟后，所有颜色恢复为默认状态。&lt;/li&gt;
&lt;li style=&quot; font-size:14pt;&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;偷看答案&lt;/span&gt;：玩家可以在游戏过程中点击“偷看答案”按钮，系统会在十秒内展示完整的答案，并在十秒后自动恢复玩家的当前输入状态。&lt;/li&gt;
&lt;li style=&quot; font-size:14pt;&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;显示答案&lt;/span&gt;：玩家可以选择在解题过程中查看完整的答案，程序将记录偷看答案的次数，并统计游戏的总用时。游戏结束时，系统会显示玩家的偷看次数以及总用时。&lt;/li&gt;&lt;/ul&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rules_back">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1110</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
